web/skins/classic/views/class/montage_class.php

```diff
@@ -576,7 +576,16 @@ class="monitorStream imageFeed"
     } else {
       //!!! IMPORTANT It is necessary to check the file with the event! It may not be there due to a failure. But something needs to be displayed!
       if ($Event->file_exists() && $Event->file_size() > 0) {
-        $fid = 1;
+        $Event = ZM\Event::find_one(array('Id'=>$eid));
```

Comment:
You already have an event object, why are you loading it again?

Reply:
My head is full of holes...
Thanks. Fixed it.
I'm busy today and probably not very attentive :(
Sorry.